src/composables/useFlaw.ts

```diff
@@ -3,12 +3,18 @@ import { ref } from 'vue';
 import type { ZodFlawType } from '@/types/zodFlaw';
 
 const flaw = ref<ZodFlawType>(blankFlaw());
+const relatedFlaws = ref<ZodFlawType[]>([]);
+
+function resetFlaw() {
+  flaw.value = blankFlaw();
+  relatedFlaws.value = [];
+}
 
 export function useFlaw() {
-  return flaw;
+  return { flaw, relatedFlaws, blankFlaw, resetFlaw };
```

Comment:
I think we could leave `blankFlaw` outside of the `useFlaw` composable as it is returning a constant value that does not modifies `flaw` in any way

Reply:
Ah these are reversed!

Reply:
I had this same thought